### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 20urc3**
Re-structure the first page to be a bit more appealing
- Regrouping getting started and fuzzer example in one sub-section
- Separate installation guide in a different sub-section
- Moving 'Why LibAFL'' further in the page
- Removing spacing/newline to group info together

### CODE REVIEW

**Comment by domenukk** [comment]
You removed the `## Resources` section (?)

**Comment by domenukk** [comment]
Ah you renamed it.. why not start with why libafl?

**Comment by 20urc3** [comment]
- I merged resource with quick start, feels like it is a bit more appealing for people to see examples and tutorials together
- The why LibAFL section is a bit verbose and more "in-depth" so I felt like behind after the *catchy* content was more relevant. The ideal scenario in my opinion would be to re-structure overview and why libafl to be one (a bit shorter) section (I  can do that today)

**Comment by domenukk** [comment]
I mean it's at the top so people who browse here the first time know why to stay...

**Comment by 20urc3** [comment]
@domenukk understandable. I'll push new version with overview\whyLibAFL merged later today

**Comment by 20urc3** [comment]
Feels like this version is clean. Lmk your opinion

File: README.md [hunk 1]
```diff
@@ -24,103 +26,63 @@ feel free to add an AST-based input for structured fuzzing, and more.
...
-```
-
-2. Clone the LibAFL repository with
-
++ `SanitizerCoverage`, in [libafl_targets](./libafl_targets)
++ `Frida`, in [libafl_frida](./libafl_frida)
++ `QEMU` user-mode and system mode, including hooks for emulation, in [libafl_qemu](./libafl_qemu)
++ `TinyInst`, in [libafl_tinyinst](./libafl_tinyinst) by [elbiazo](https://github.com/elbiazo)
+
+## Building and installing
```
Comment (domenukk) [inline README.md:35]: Title case: `Building and Installing`

File: README.md [hunk 2]
```diff
@@ -24,103 +26,63 @@ feel free to add an AST-based input for structured fuzzing, and more.
...
+    - We use cargo-make to build the fuzzers in `fuzzers/` directory. You can install it with `cargo install cargo-make`
+ 
+#### Clone the LibAFL repository with
 ```sh
 git clone https://github.com/AFLplusplus/LibAFL
 ```
-
-3. Build the library using
-
+#### Build the library using
```
Comment (domenukk) [inline README.md:50]: Title case some more

File: README.md [hunk 3]
```diff
@@ -13,7 +13,16 @@ LibAFL is written and maintained by
...
+LibAFL is a collection of reusable pieces of fuzzers, written in Rust. It is fast, multi-platform, no_std compatible, and scales over cores and machines. It offers a main crate that provide building blocks for custom fuzzers, [libafl](./libafl), a library containing common code that can be used for targets instrumentation, [libafl_targets](./libafl_targets), and a library providing facilities to wrap compilers, [libafl_cc](./libafl_cc). 
+
+
+LibAFL offers integrations with popular instrumentation frameworks. At the moment, the supported backends are:
++ `SanitizerCoverage`, in [libafl_targets](./libafl_targets)
++ `Frida`, in [libafl_frida](./libafl_frida)
++ `QEMU` user-mode and system mode, including hooks for emulation, in [libafl_qemu](./libafl_qemu)
++ `TinyInst`, in [libafl_tinyinst](./libafl_tinyinst) by [elbiazo](https://github.com/elbiazo)
 
 LibAFL gives you many of the benefits of an off-the-shelf fuzzer, while being completely customizable.
```
Comment (domenukk) [inline README.md:27]: I think we should keep this as extra section, the new Overview section is too big IMHO

Comment (20urc3) [reply README.md:28]: Done!

**Comment by 20urc3** [comment]
Should be good for you now @domenukk

File: README.md [hunk 4]
```diff
@@ -24,103 +13,74 @@ feel free to add an AST-based input for structured fuzzing, and more.
...
-The best-tested fuzzer is [`./fuzzers/inprocess/libfuzzer_libpng`](./fuzzers/inprocess/libfuzzer_libpng), a multicore libfuzzer-like fuzzer using LibAFL for a libpng harness.
-
-## Resources
-
-+ [Installation guide](./docs/src/getting_started/setup.md)
-
-+ [Online API documentation](https://docs.rs/libafl/)
-
-+ The LibAFL book (WIP) [online](https://aflplus.plus/libafl-book) or in the [repo](./docs/src/)
+### Ressources
```
[the remark this answers is not in the document]
Comment (20urc3) [reply README.md]: @tokatoka Thanks for the merge! The latest version correspond more to what Dom asked :-)

**Pull request merged**